armedilla_alliyah_samantha/script.js

```diff
@@ -1,62 +1,69 @@
-function validateInput() {
+function validate_input() {
```

Comment:
Function names should be written in camel case. For example:
❌ function validate_input () {
✅ function validateInput() {

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g25d183194fb_0_0

Reply:
Also in line 16 & 21.